metadata/version.go

```diff
@@ -8,7 +8,7 @@
 package metadata
 
 //Version of Maestro
-var Version = "8.1.1"
+var Version = "8.2.1"
```

Comment:
Why not 8.2.0?

Reply:
8.2.0 was released already. Forgot to change the metadata =P